distro_data.json

```diff
@@ -518,18 +518,16 @@
             "eol": "2017-05"
         },
         "versions": {
-            "kernel": "4.1",
-            "GCC": [
-                "4.8",
-                "5.2"
-            ],
-            "clang": "3.7",
-            "glib": "2.44",
+            "pixman": "0.32.6",
```

Comment:
We don't need this level of precision, I'd rather not touch anything when x.y match in x.y.z.

Reply:
ok, so removing `.z` everywhere. I figured you probably don't need it, but I wrote a script that crawled our build service and was just too lazy to manually remove all of them :-)

Reply:
That would actually be useful, if that matches what's in those releases.